app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -54,6 +54,11 @@ def update
 
     private
 
+    def confirm_ssn_step_complete
```

Comment:
We've been putting these methods that check flow state into IdvStepConcern. I think that would be a better place for `confirm_ssn_step_complete`.

It's more debatable for `delete_pii`, but we have `pii_from_doc` in there, so it could move too. We're eventually going to consolidate the flow_session code and use idv_session.

Reply:
Thank you! I moved `confirm_ssn_step_complete` into `IdvStepConcern` in 47914a6e0c921e5c4517c63b16b6e51fc1a274db

I think I'll leave `delete_pii` where it is, because it's only used by `move_applicant_to_idv_session`, also in the `Idv::VerifyInfoConcern`.